spicetify.go

```diff
@@ -251,6 +251,15 @@ func main() {
 			cmd = exec.Command(ex, strings.Join(commands[:], " "))
 			utils.CmdScanner(cmd)
 		}
+
+		if spotStat.IsBackupable() {
+			utils.PrintInfo("spicetify is already up-to-date! If you ran this command because spicetify disappeared after Spotify updated, we'll attempt to fix it for you right now.")
```

Comment:
```suggestion
			utils.PrintInfo("There was an update to Spotify, but Spicetify is still up-to-date, so we'll try to reapply Spicetify for you!")
```
What do you think?

Reply:
We don't know if there was an update, i think we can reword the message I made but not change the meaning of it, since your change changes it in some way

Reply:
You think? I'm still saying that Spicetify is no longer applied, but it is up to date 🤔

Reply:
Sure, but I always use the word "if". You're making a point in this message even if it could be not true. Also, we're attempting to fix it because we have no way if it will work or not